frontend/src/components/Buttons/PlaybackSettingsButton.vue

```diff
@@ -1,143 +1,98 @@
+          </v-col>
+          <v-col :cols="8">
+            <v-select density="comfortable" hide-details disabled />
+          </v-col>
+        </v-row>
+        <v-row align="center">
+          <v-col :cols="4">
+            <label>{{ $t('audio') }}</label>
+          </v-col>
+          <v-col :cols="8">
+            <media-stream-selector
+              v-if="playbackManager.currentItemAudioTracks"
+              :media-streams="playbackManager.currentItemAudioTracks"
+              type="Audio"
+              :default-stream-index="playbackManager.currentAudioStreamIndex"
+              @input="playbackManager.currentAudioStreamIndex = $event" />
+          </v-col>
+        </v-row>
+        <v-row v-if="!$vuetify.display.smAndUp" align="center">
+          <v-col :cols="4">
+            <label>{{ $t('subtitles') }}</label>
+          </v-col>
+          <v-col :cols="8">
+            <media-stream-selector
+              v-if="playbackManager.currentItemSubtitleTracks"
+              :media-streams="playbackManager.currentItemSubtitleTracks"
+              type="Subtitle"
+              :default-stream-index="playbackManager.currentSubtitleStreamIndex"
+              @input="playbackManager.currentSubtitleStreamIndex = $event" />
+          </v-col>
+        </v-row>
+        <v-row align="center">
+          <v-col :cols="4">
+            <label>{{ $t('speed') }}</label>
+          </v-col>
+          <v-col :cols="8">
+            <v-select density="comfortable" disabled hide-details />
+          </v-col>
+        </v-row>
+        <v-row align="center">
+          <v-col :cols="4">
+            <label>{{ $t('stretch') }}</label>
+          </v-col>
+          <v-col :cols="8" class="text-right">
+            <v-switch v-model="stretched" color="primary" hide-details />
+          </v-col>
+        </v-row>
+      </v-card-text>
     </v-card>
   </v-menu>
 </template>
 
-<script lang="ts">
-import { defineComponent } from 'vue';
-import { playbackManagerStore } from '@/store';
+<script lang="ts" setup>
+import { computed } from 'vue';
+import { useVModel } from '@vueuse/core';
+import { playbackManagerStore, videoPlayerStore } from '@/store';
 
-export default defineComponent({
-  props: {
-    nudgeTop: {
-      type: [Number, String],
-      default: 0
-    },
-    stretchProp: Boolean
-  },
-  setup() {
-    const playbackManager = playbackManagerStore();
+const props = defineProps<{
+  modelValue: boolean;
+}>();
+const emit = defineEmits(['update:modelValue']);
 
-    return {
-      playbackManager
-    };
-  },
-  data() {
-    return {
-      menu: false,
-      stretch: this.stretchProp
-    };
-  },
-  watch: {
-    stretchProp(value) {
-      this.stretch = value;
-    },
-    stretch(value) {
-      this.$emit('stretch', value);
-    }
+const menuModel = useVModel(props, 'modelValue', emit);
+
+const playbackManager = playbackManagerStore();
+const videoPlayer = videoPlayerStore();
```

Comment:
I would rename this store to ``playerElementStore`` instead

Reply:
Done